cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,34 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Subsection Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="visible" aria-described-by"visible_description">
```

Comment:
"aria-described-by" is not a valid ARIA attribute. It is "aria-describedby". You can find this sort of error using Chrome's built-in accessibility Audit (in Chrome developer tools).

Also there is a typo in line 7 in that there is no equal sign.

I also don't understand the usage of ul/li with radio buttons inside of li's. I'll open a JIRA ticket for the issues. I imagine this HTML was likely copied from somewhere else, just as another PR has now copied this!